features/support/env.rb

```diff
@@ -103,7 +103,7 @@
 end
 
 # Force deprecations to raise an exception.
-ActiveSupport::Deprecation.behavior = :raise
+ActiveSupport::Deprecation.behavior = ActiveAdmin::Dependency.rails?('< 5.1.x') ? :raise : :stderr # FIXME: need database_cleaner 1.6.0.
```

Comment:
1.6.0 is not released yet. We can revert Gemfile to `gem 'database_cleaner',  github: 'DatabaseCleaner/database_cleaner'` as it was last summer after Rails 5.0.